modular_skyrat/modules/alternative_job_titles/code/job.dm

```diff
@@ -0,0 +1,20 @@
+// ALTERNATIVE_JOB_TITLES
+
+/datum/controller/subsystem/job/proc/setup_alt_job_items(mob/living/carbon/human/equipping, datum/job/job, client/player_client)
```

Comment:
If you could add some documentation to that proc, since it's a new one you're adding, that'd be really great. There's also a neat multiline comment helper extension for VSC that'll make your life easier, I think it's recommended by the repo too.